sdk/python/kfp/_client.py

```diff
@@ -74,6 +74,8 @@ def camel_case_to_snake_case(name):
 KF_PIPELINES_DEFAULT_EXPERIMENT_NAME = 'KF_PIPELINES_DEFAULT_EXPERIMENT_NAME'
```

Comment:
@Ark-kun I think we could move these environment variable settings to the file-based settings as well (of course in a separate PR later). WDYT?

Reply:
JFYI: This variables are used by the sample tests.